Sources/NIO/Bootstrap.swift

```diff
@@ -364,8 +385,121 @@ public final class ServerBootstrap {
             }
         }
     }
+    
+    /// A channel option which can be applied to bootstrap using shorthand notation.
+    /// - See: ServerBootstrap.serverOptions(_ options: [ShorthandBootstrapOption])
+    public struct ShorthandServerBootstrapOption {
+        private let data: ShorthandServerOption
+        
+        private init(_ data: ShorthandServerOption) {
+            self.data = data
+        }
+        
+        /// Apply the contained option to the supplied ServerBootstrap
+        /// - Parameter serverBootstrap: bootstrap to apply this option to.
+        /// - Returns: the modified bootstrap (currently the same one mutated)
+        func applyOption(to serverBootstrap: ServerBootstrap) {
+            data.applyOption(to: serverBootstrap)
+        }
+        
+        fileprivate enum ShorthandServerOption {
+            case reuseAddr(Bool)
+            case autoRead(Bool)
+            case backlog(Int32)
+            
+            func applyOption(to serverBootstrap: ServerBootstrap) {
+                switch self {
+                case .autoRead(let value):
+                    _ = serverBootstrap.serverChannelOption(ChannelOptions.autoRead, value: value)
+                case .reuseAddr(let value):
+                    _ = serverBootstrap.serverChannelOption(ChannelOptions.socketOption(.reuseaddr),
+                                                            value: value ? 1 : 0)
+                case .backlog(let value):
+                    _ = serverBootstrap.serverChannelOption(ChannelOptions.backlog, value: value)
+                }
+            }
+        }
+    }
+    
+    /// A channel option which can be applied to bootstrap using shorthand notation.
+    /// - See: ServerBootstrap.serverOptions(_ options: [ShorthandBootstrapOption])
```

Comment:
This documentation should be for the child channel

Reply:
Done.

Reply:
Renamed to the suggested, .maximumUnacceptedConnectionBacklog